packages/extension-chakra-storefront/src/pages/account/index.jsx

```diff
@@ -200,36 +193,40 @@ const Account = () => {
-                        {navLinks.map((link) => {
-                            const LinkIcon = link.icon
-                            return (
-                                <Button
-                                    key={link.name}
-                                    as={Link}
-                                    to={`/account${link.path}`}
-                                    useNavLink={true}
-                                    variant="menu-link"
-                                    leftIcon={<LinkIcon boxSize={5} />}
-                                >
-                                    {formatMessage(messages[link.name])}
-                                </Button>
-                            )
-                        })}
+                    <Flex gap="0" as="nav" data-testid="account-detail-nav" direction="column">
+                        <List.Root variant="plain" as="ul">
+                            {navLinks.map((link) => {
+                                const LinkIcon = link.icon
+                                return (
+                                    <List.Item key={link.name}>
+                                        <Button
+                                            asChild
+                                            variant="menu-link"
+                                            css={styles.menuAccountLink}
+                                        >
+                                            <Link to={`/account${link.path}`} useNavLink={true}>
+                                                <LinkIcon boxSize="5" mr="2" />
+                                                {formatMessage(messages[link.name])}
+                                            </Link>
+                                        </Button>
+                                    </List.Item>
+                                )
+                            })}
+                        </List.Root>
+
                         <LogoutButton onClick={onSignoutClick} />
                     </Flex>
                 </Stack>
-
                 <Switch>
-                    <Route exact path={path}>
-                        <AccountDetail />
-                    </Route>
-                    <Route exact path={`${path}/wishlist`}>
-                        <AccountWishlist />
-                    </Route>
-                    <Route exact path={`${path}/addresses`}>
-                        <AccountAddresses />
-                    </Route>
+                    {/*<Route exact path={path}>*/}
+                    {/*    <AccountDetail />*/}
+                    {/*</Route>*/}
+                    {/*<Route exact path={`${path}/wishlist`}>*/}
+                    {/*    <AccountWishlist />*/}
+                    {/*</Route>*/}
+                    {/*<Route exact path={`${path}/addresses`}>*/}
+                    {/*    <AccountAddresses />*/}
+                    {/*</Route>*/}
```

Comment:
The account address changes have been merged in #2719 so I think we can enable it here

Reply:
Gotcha. I've enabled it and fixed some old `spacing` to `gap`